ssas/service/admin/api.go

```diff
@@ -0,0 +1,41 @@
+package admin
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/CMSgov/bcda-app/ssas"
+	"net/http"
+)
+
+func createSystem(w http.ResponseWriter, r *http.Request) {
+	type system struct {
+		GroupID    string `json:"group_id"`
+		ClientID   string `json:"client_id"`
+		ClientName string `json:"client_name"`
+		ClientURI  string `json:"client_uri"`
+		Scope      string `json:"scope"`
+		PublicKey  string `json:"public_key"`
+	}
+
+	sys := system{}
+	if err := json.NewDecoder(r.Body).Decode(&sys); err != nil {
+		http.Error(w, "Invalid request body", http.StatusBadRequest)
+		return
+	}
+
+	creds, err := ssas.RegisterSystem(sys.ClientID, sys.ClientName, sys.ClientURI, sys.GroupID, sys.Scope, sys.PublicKey)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Could not create system. Error: %s", err), http.StatusInternalServerError)
```

Comment:
For many of the possible errors returned by ssas.RegisterSystem(), http.StatusBadRequest might be better.